velox/expression/Expr.cpp

```diff
@@ -40,6 +41,20 @@ DEFINE_bool(
     "Whether to overwrite queryCtx and force the "
     "use of simplified expression evaluation path.");
 
+DEFINE_bool(
+    experimental_velox_save_input_on_fatal_signal,
+    false,
+    "This is an experimental flag that, when set to true, "
```

Comment:
nit: Maybe write as follows: This is an experimental flag only to be used for debugging purposes . If set , it serializes...